activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,76 @@
+*   Remove deprecated class `ActiveSupport::Concurrency::Latch`
+
+    *Andrew White*
+
+*   Remove deprecated separator argument from `parameterize`
+
+    *Andrew White*
+
+*   Remove deprecated method `Numeric#to_formatted_s`
+
+    *Andrew White*
+
+*   Remove deprecated method `alias_method_chain`
+
+    *Andrew White*
+
+*   Remove deprecated constant `MissingSourceFile`
+
+    *Andrew White*
+
+*   Remove deprecated methods `Module.qualified_const_defined?`,
+    `Module.qualified_const_get` and `Module.qualified_const_set`
+
+    *Andrew White*
+
+*   Remove deprecated `:prefix` option from `number_to_human_size`
+
+    *Andrew White*
+
+*   Remove deprecated method `ActiveSupport::HashWithIndifferentAccess.new_from_hash_copying_default`
+
+    *Andrew White*
+
+*   Remove deprecated method `Module.local_constants`
+
+    *Andrew White*
+
+*   Remove deprecated file `active_support/core_ext/time/marshal.rb`
```

Comment:
@bogdan not sure what you mean - it was an empty file that was kept around so that it didn't break people's require statements so no methods were removed.